src/common/config_opts.h

```diff
@@ -538,6 +538,7 @@ OPTION(osd_heartbeat_addr, OPT_ADDR, entity_addr_t())
 OPTION(osd_heartbeat_interval, OPT_INT, 6)       // (seconds) how often we ping peers
 OPTION(osd_heartbeat_grace, OPT_INT, 20)         // (seconds) how long before we decide a peer has failed
 OPTION(osd_heartbeat_min_peers, OPT_INT, 10)     // minimum number of peers
+OPTION(osd_heartbeat_use_min_delay_socket, OPT_BOOL, false)  // set SO_PRIORITY of the sockets as 6(high) if true
```

Comment:
Oh, please let's leave it off until we've figured out a good canary to test
it on for a while. :)

On Mon, Jan 12, 2015 at 8:14 AM, Sage Weil <notifications@github.com> wrote:

> In src/common/config_opts.h
> <https://github.com/ceph/ceph/pull/3353#discussion-diff-22800126>:
>
> > @@ -538,6 +538,7 @@ OPTION(osd_heartbeat_addr, OPT_ADDR, entity_addr_t())
> >  OPTION(osd_heartbeat_interval, OPT_INT, 6)       // (seconds) how often we ping peers
> >  OPTION(osd_heartbeat_grace, OPT_INT, 20)         // (seconds) how long before we decide a peer has failed
> >  OPTION(osd_heartbeat_min_peers, OPT_INT, 10)     // minimum number of peers
> > +OPTION(osd_heartbeat_use_min_delay_socket, OPT_BOOL, false)  // set SO_PRIORITY of the sockets as 6(high) if true
>
> maybe in a separate patch, actually, so that the first patch can be
> backported
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ceph/ceph/pull/3353/files#r22800126>.
>